lib/ethpy/ethpy/hyperdrive/api/_contract_calls.py

```diff
@@ -400,14 +400,14 @@ async def _async_add_liquidity(
     # before calling smart contract transact
     # Since current_pool_state.block_number is a property, we want to get the static block here
     current_block = interface.current_pool_state.block_number
-    _ = smart_contract_preview_transaction(
-        interface.hyperdrive_contract,
-        agent_checksum_address,
-        "addLiquidity",
-        *fn_args,
-        block_number=current_block,
-        read_retry_count=interface.read_retry_count,
-    )
+    # _ = smart_contract_preview_transaction(
```

Comment:
just delete this code if you don't want it

Reply:
Also, what's going on here? It looks like we did the preview call but then never used it? @sentilesdal do you have any insight as to how this preview_transaction command came to be & what it was intended for? I assume since we were just dumping the result into `_` Mihai is right to remove it entirely.